misk-clustering-dynamodb/src/main/kotlin/misk/clustering/dynamo/DynamoClusterWatcherTask.kt

```diff
@@ -85,8 +87,8 @@ internal class DynamoClusterWatcherTask @Inject constructor(
           members.add(Member(item.name!!, "invalid-ip"))
         }
       }
-
-      dynamoCluster.update(members)
+      cluster.clusterChanged(membersBecomingReady = members, membersBecomingNotReady = prevMembers - members)
```

Comment:
do you know how `membersBecomingNotReady` are used? I am not sure it is correct to say that a previous member that that no longer exists is not ready. should we just always set `membersBecomingNotReady` to empty set?

Reply:
it removes it from `readyMembers` otherwise it's still considered `ready`. Yea the variable names might be confusing in terms of how we're tracking it now in dynamoDB. But we should still remove it from the cluster, if it's not in the table anymore